### ISSUE TITLE
Branch 11 showing directory of WEB-INF instead of welcome.html

### ISSUE THREAD

**Issue by rafaellago** [jetty-project/embedded-servlet-server#183]
As the title says, just cloned the branch 11.0.x and ran `mvn clean compile exec:exec` server started, no problems.

```
WebResourceBase (Using /src/main/webapp/ Path) file:///home/lago/code/embedded-servlet-server/src/main/webapp/WEB-INF/
Using BaseResource: file:///home/lago/code/embedded-servlet-server/src/main/webapp/WEB-INF/
2023-08-29 20:57:22.216:INFO :oejs.Server:main: jetty-11.0.15; built: 2023-04-11T18:37:53.775Z; git: 5bc5e562c8d05c5862505aebe5cf83a61bdbcb96; jvm 17.0.8+7-Debian-1deb12u1
2023-08-29 20:57:22.405:INFO :oejw.StandardDescriptorProcessor:main: NO JSP Support for /, did not find org.eclipse.jetty.jsp.JettyJspServlet
2023-08-29 20:57:22.415:INFO :oejss.DefaultSessionIdManager:main: Session workerName=node0
2023-08-29 20:57:22.444:INFO :oejsh.ContextHandler:main: Started o.e.j.w.WebAppContext@15761df8{/,file:///home/lago/code/embedded-servlet-server/src/main/webapp/WEB-INF/,AVAILABLE}
2023-08-29 20:57:22.455:INFO :oejs.AbstractConnector:main: Started ServerConnector@3caeaf62{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
2023-08-29 20:57:22.463:INFO :oejs.Server:main: Started Server@4b9e13df{STARTING}[11.0 .15,sto=0] @538ms
```

But when calling the server, it just displays the tree for the WEB-INF folder.

![image](https://github.com/jetty-project/embedded-servlet-server/assets/7526047/e1dbb848-d6c7-460b-b5ed-9ca4848b9e5a)

**Comment by lachlan-roberts** [comment]
@rafaellago it looks like you have configured the base resource of your app to `/src/main/webapp/WEB-INF/` instead of `/src/main/webapp/`. The `webapp` dir should be the base directory and `WEB-INF` contained within it.

These lines from the log you posted show the misconfiguration:
```
WebResourceBase (Using /src/main/webapp/ Path) file:///home/lago/code/embedded-servlet-server/src/main/webapp/WEB-INF/
Using BaseResource: file:///home/lago/code/embedded-servlet-server/src/main/webapp/WEB-INF/
2023-08-29 20:57:22.444:INFO :oejsh.ContextHandler:main: Started o.e.j.w.WebAppContext@15761df8{/,file:///home/lago/code/embedded-servlet-server/src/main/webapp/WEB-INF/,AVAILABLE}
```

If you share how you configured your server we can help identify where the problem is.

**Comment by rafaellago** [comment]
I ran the project without any changes on it. Just cloned and ran it as is with maven exec. I have noticed that on this part in EmbedMe.java, the code is actually getting the parent of web.xml which in turn is WEB-INF.

```java
// Try the source path next
Path srcWebapp = pwd.resolve("src/main/webapp/" + webResourceRef);
if (Files.exists(srcWebapp))
{
    URI uri = srcWebapp.getParent().toUri();
    System.err.printf("WebResourceBase (Using /src/main/webapp/ Path) %s%n", uri);
    return uri;
}
```
Adding a second getParent() on the path before converting to URI seemed to solve the problem.

**Comment by joakime** [comment]
I'll update the code in these examples to use URI (or Path) properly.

**Comment by joakime** [comment]
Added mvn wrapper and fixed both `jetty-10.0.x` and `jetty-11.0.x` branches.
